test/unit/avif.js

```diff
@@ -5,7 +5,7 @@ const assert = require('assert');
 const sharp = require('../../');
 const { inputAvif, inputJpg, inputGifAnimated } = require('../fixtures');
 
-describe('AVIF', () => {
+(sharp.format.heif.input.file ? describe : describe.skip)('AVIF', () => {
```

Comment:
AVIF support is pretty heavy and not needed for everyone, so it's good to have a switch - especially in tests - that runs the tests only if it's built in.

Reply:
@RReverser Does your PR support AVIF? I thought it did but now that I [tried it out](https://github.com/lovell/sharp/pull/3522#issuecomment-1667857371) I'm not seeing AVIF.

The reason I ask is because Squoosh does support AVIF and I want to replace Squoosh with Sharp.

Reply:
You got me worried for a second, but just checked and it does, yeah: https://stackblitz.com/edit/node-9c65qs?file=shark.avif

Reply:
> I'm not seeing AVIF

Did you mean you checked via `console.log(sharp.format)` or something like that perhaps? If so, that's because it's called HEIF there (which includes AVIF as it's a container for the latter).

Reply:
Ah yes, I did `console.log(require('sharp'))` and was incorrectly looking at formats.

You're right, AVIF works. Thanks!